frontend/packages/dev-console/src/components/health-checks/AddHealthChecks.tsx

```diff
@@ -92,19 +92,19 @@ const AddHealthChecks: React.FC<FormikProps<FormikValues> & AddHealthChecksProps
           </>
         }
       />
-      <div className="odc-add-health-checks__body">
-        <p>
-          {t('devconsole~Health checks for')} &nbsp;
-          <ResourceLink
-            kind={referenceFor(resource)}
-            name={name}
-            namespace={namespace}
-            title={name}
-            inline
-          />
-        </p>
-        <Form onSubmit={!viewOnly ? handleSubmit : undefined}>
-          <div>
+      <Form onSubmit={!viewOnly ? handleSubmit : undefined}>
+        <div className="odc-add-health-checks__body">
+          <p>
+            {t('devconsole~Health checks for')} &nbsp;
+            <ResourceLink
+              kind={referenceFor(resource)}
+              name={name}
+              namespace={namespace}
+              title={name}
+              inline
+            />
+          </p>
```

Comment:
Nit: This should probably be using `<Trans>...</Trans>`. Translators should always be given full context as not all languages keep the same sentence structure. Sometimes the noun is first, sometimes it's last.